source/nodejs/adaptivecards/src/enums.ts

```diff
@@ -123,7 +123,8 @@ export enum InputTextStyle {
     Text,
     Tel,
     Url,
-    Email
+    Email,
+    Password
```

Comment:
It's pretty cool that the type maps directly to the HTML type !

I think we need some schema change on the `TextInput` class in order to version this properly, though... here's an example of a versioned property value in `ChoiceSetInput`:

https://github.com/microsoft/AdaptiveCards/blob/bf65cddc824b64b95a0acd635504b314ae3a94ad/source/nodejs/adaptivecards/src/card-elements.ts#L3128-L3136

this info will help the designer give accurate warnings and allow the renderer to know whether to honor the `password` value when rendering in lower versions...